app/test_main.py

```diff
@@ -1,3 +1,51 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,expected_age_values",
+        [
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="must return 0 for both animals"
+            ),
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="must add 1 year in both cases"
+            ),
+            pytest.param(
+                23,
+                23,
+                [1, 1],
+                id="must add only 1 year in both cases"
+            ),
+            pytest.param(
+                24,
+                24,
+                [2, 2],
+                id="must add 2 years in both cases"
+            ),
+            pytest.param(
+                28,
+                28,
+                [3, 2],
+                id="must be 3 years for cat age and 2 years for dog age"
+            ),
+            pytest.param(
+                65,
+                65,
+                [12, 10],
+                id="must return the correct age for every next years"
```

Comment:
not all edge cases covered, error handling not covered - please fix as per the task checklist

Reply:
By error handling, did you mean incorrect values ​​at the function input? For example negative values ​​or data type error? If so, is it possible to change the code in the main file, for example, to add a raise condition, or should this only be done in tests?

Reply:
negative numbers will not produce error - you must include negative and really large numbers in these params as per checklist. Errors may be triggered only by wrong data types

Reply:
> negative numbers will not produce error - you must include negative and really large numbers in these params as per checklist. Errors may be triggered only by wrong data types

I see, so there is no problem with negative numbers, because the function doesn't have a condition that the numbers must be greater than zero. So, can I add such test cases and write the result to expected, even if it is incorrect (value of age less than zero)?